### ISSUE TITLE
283171697 Get dreaded red message when I get to the end of a file (Geoff)

### ISSUE THREAD

**Issue by FreeUKCen** [FreeUKGen/FreeCENMigration#1615]
Issue reported by **GeoffJ** at 2023-12-22 10:48:17 UTC
Time: 2023-12-22T10:46:45+00:00
Session ID: 2c9ea0c47b901cd9ab645b0bfe5d4c0d
Problem Page URL: [/freecen1_vld_entries/propagate_pob?id=5904bff4e9379091b1c36b2e&locale=en&propagation_fields=Alternative](/freecen1_vld_entries/propagate_pob?id=5904bff4e9379091b1c36b2e&locale=en&propagation_fields=Alternative)
Previous Page URL: [https://www.freecen.org.uk/freecen1_vld_entries/edit_pob?id=5904bff4e9379091b1c36b2e&locale=en](https://www.freecen.org.uk/freecen1_vld_entries/edit_pob?id=5904bff4e9379091b1c36b2e&locale=en)
Reported Issue:
When I propagate the last file entry in a vld re-validation I get the dreaded red message instead of cycling back to the file list.
Geoff
    

![Screenshot](https://www.freecen.org.uk)

**Comment by AnneV-Learn** [comment]
@geoffj-FUG I have tried to recreate this issue but have not been able to. I tried propagation in various scenarios: - 

- no more invalid records after successful propagation; 

- propagation on the last invalid record in the file but where other invalid records still exist (ie some have been skipped);

- propagation on last record in the list when there is only 1 invalid record prior to propagation.

It behaved as expected  for each of these ie no 'red error message'  appeared.
